packages/web-react/src/components/UNSTABLE_EmptyState/README.md

```diff
@@ -2,3 +2,79 @@
 
 ⚠️ This component is UNSTABLE. It may significantly change at any point in the future.
 Please use it with caution.
+
```

Comment:
Maybe something like this: `Indicates when there’s no data to show, like when a search has no results.` ?

Reply:
I actually used the text from Jobs DS